pkg/sensors/tracing/generickprobe.go

```diff
@@ -1148,7 +1155,11 @@ func handleGenericKprobe(r *bytes.Reader) ([]observer.Event, error) {
 		return nil, fmt.Errorf("Failed to match id")
 	}
 
-	return handleMsgGenericKprobe(&m, gk, r)
+	ret, err := handleMsgGenericKprobe(&m, gk, r)
```

Comment:
Here just making sure you don't want to return the error if something happens? It will be overridden by the next call if the customHandler is not nil.

Reply:
yap, it's intentional. The handler can decide what to do both with the event and the error.